move/squid/sources/squid/squid.move

```diff
@@ -67,6 +69,29 @@ entry fun give_deep(self: &mut Squid, deep: Coin<DEEP>) {
     self.value_mut!(b"give_deep").give_deep(deep);
 }
 
+entry fun allow_function(
+    self: &mut Squid,
+    _: &OwnerCap,
+    version: u64,
+    function_name: String,
+) {
+    self.value_mut!(b"allow_function").allow_function(version, function_name);
+}
+
+entry fun disallow_function(
+    self: &mut Squid,
+    _: &OwnerCap,
+    version: u64,
+    function_name: String,
+) {
+    self
+        .value_mut!(b"disallow_function")
+        .disallow_function(version, function_name);
+}
+
+entry fun withdraw<T>(self: &mut Squid, _: &OwnerCap, amount: u64, ctx: &mut TxContext) {
```

Comment:
best to avoid including unrelated functionality in the PR, easier to miss during review